app/jobs/appointment_cancelled_notification_job.rb

```diff
@@ -0,0 +1,13 @@
+class AppointmentCancelledNotificationJob < ActiveJob::Base
+  queue_as :default
+
+  MESSAGE = 'cancelled'.freeze
```

Comment:
I think these message constants should be defined in the `CustomerUpdateActivity`. Any overriding reason they're defined here?